resources/public/js/agent.js

```diff
@@ -177,7 +177,10 @@ $(document).ready(function() {
       } else {
         location.reload();
       }
-    }, "json");
+    }, "json").fail(function (operationResult){
```

Comment:
Here and everywhere. 
```suggestion
    }, "json").fail(function (operationResult) {
```

Reply:
Fixed all occurences.